uuid_source_test.go

```diff
@@ -0,0 +1,62 @@
+package uuid
+
+import (
+	"testing"
+	"strings"
+	
+	regen "github.com/zach-klippenstein/goregen"
+)
+
+func TestUuidSources(t *testing.T) {
+	
+	myString, _ := regen.Generate("[a-zA-Z]{1000}")
```

Comment:
Rather than depend on another external package (creating another dependency) why not just make a very simple reader that returns a deterministic set based on some seed.  For example:

```
type reader struct {
	n int
	c int
}

func (r *reader) Read(buf []byte) (int, error) {
	// Make an uninitialized reader just return sequential bytes.
	 if r.n == 0 {
		r.n = 1
	}
	c, n := r.c, r.n
	for i := range buf {
		c = (c + n) * n
		buf[i] = byte(c)
	}
	r.c = c
	return len(buf), nil
}
```